ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -1265,3 +1265,58 @@ def test_check_ad_idrange_is_migrated(self):
             ["ipa", "idrange-show", ad_domain_name + "_id_range"]
         )
         assert cmd1.stdout_text == cmd2.stdout_text
+
+
+class TestIPAMigratewithBackupRestore(IntegrationTest):
+    """
+    Tests for ipa-migrate tool with backup files.
+    """
+    num_replicas = 1
+    topology = "line"
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=True, setup_kra=True)
+        prepare_ipa_server(cls.master)
+        tasks.install_master(cls.replicas[0], setup_dns=True, setup_kra=True,
+                             extra_args=['--allow-zone-overlap'])
+
+    def test_ipa_migrate_stage_mode(self):
+        """
+        This test checks ipa-migrate with LDIF file
+        from backup of remote server is successful.
+        """
+        ERR_MSG = (
+            "error: change collided with another change"
+        )
+        dashed_domain_name = self.master.domain.realm.replace(
+            ".", '-'
+        )
+        DB_LDIF_FILE = '{}-userRoot.ldif'.format(
+            dashed_domain_name
+        )
+        SCHEMA_LDIF_FILE = '{}''/config_files/schema/99user.ldif'.format(
+            dashed_domain_name)
+        CONFIG_LDIF_FILE = '{}''/config_files/dse.ldif'.format(
+            dashed_domain_name)
+        param = [
+            '-n', '-g', CONFIG_LDIF_FILE, '-m', SCHEMA_LDIF_FILE,
+            '-f', DB_LDIF_FILE
+        ]
+        tasks.kinit_admin(self.master)
+        tasks.kinit_admin(self.replicas[0])
+        backup_path = tasks.get_backup_dir(self.replicas[0])
```

Comment:
The test is doing the backup on replicas[0] and then call ipa-migrate on replicas[0] with the backup files.
The scenario should do a backup on master, copy the files to replicas[0] and then call ipa-migrate on replicas[0] with the files.